src/sbt-test/sbt-scalafix/scalafixOnCompile/build.sbt

```
@@ -21,3 +21,8 @@ lazy val rewrite = project
     addCompilerPlugin(scalafixSemanticdb)
   )
   .settings(scalafixConfigSettings(IntegrationTest): _*)
+
+lazy val args = project
```

what does args refer to? what about
```suggestion
lazy val triggered = project
```